dist/components/alert/_alert.scss

```diff
@@ -18,8 +18,8 @@ $alert__success-color: if(variable-exists(success-color), $success-color, green)
 $alert__error-color: if(variable-exists(error-color), $error-color, red) !default;
 
 
-// Alert Root
-// ----------
+// Alert root
```

Comment:
Not a big deal, but these could probably have stayed capitilized just fine, expecially since they are headings.

Anyway, nothing to worry about as far as this PR is really concerned.